src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -177,5 +237,37 @@ private static void OnUIElementKeyUp(object sender, KeyRoutedEventArgs e)
 				_ => default,
 			};
 		}
+
+#if __ANDROID__
+		private static ImeAction GetImeActionFromReturnType(ReturnType returnType)
+		{
+			switch (returnType)
+			{
+				case ReturnType.Go: return ImeAction.Go;
+				case ReturnType.Search: return ImeAction.Search;
+				case ReturnType.Send: return ImeAction.Send;
+				case ReturnType.Done: return ImeAction.Done;
+				case ReturnType.Default:
+				case ReturnType.Next:
+				default: return ImeAction.Next;
```

Comment:
Should the default be ImeAction.Next or a different ImeAction?

Reply:
same as ios,
default/else => default
next => next